data/test/scenarios/test_role_types.cfg

```diff
@@ -75,6 +98,16 @@
     [/event]
 )}
 
+#####
+# API(s) being tested: [role],[have_unit]role=
+##
+# Actions:
+# Create a role with no types assigned.
```

Comment:
A role isn't an object, it's not something that can be created.

Try to assign a role, but using a filter that doesn't match any unit (there are only 2 sides in this test).